### ISSUE TITLE
"The skip-link target should exist and be focusable" should be considered 'wcag21a' criteria

### ISSUE THREAD

**Issue by harryhmc** [dequelabs/axe-core#2268]
When using WAVE to manually test our web app, we picked up on a violation (described in the title) that was not being caught by AXE -- or rather, was only being considered best practice. The rule in question represents the [WCAG success criterion 2.1.1](https://www.w3.org/TR/WCAG21/#x2-1-1-keyboard) and thus should be fulfilled in order to achieve a 2.1a / 2.1aa rating.

For the sake of reproducibility, the offending element was a skip link that wasn't correctly targeting the main section:
<code>\<a data-test="skip-to-main" href="#maincontent">Skip to main content\</a\></code>

Expectation: "The skip-link target should exist and be focusable" should be tagged as 'wcag21a'

Actual: Said rule is tagged with 'best-practice' and is not considered a violation

Motivation: We currently use axe, along with [cypress](https://github.com/cypress-io/cypress), as part of our automated accessibility testing. While in this case we found the issue through manual testing we would rather not give up on also using axe in our automated testing to support our project.

<pre><code>axe-core version: 3.5.3
axe-webdriver, extension or other integration version: axe browser extension v.4.5.1

Browser and Assistive Technology versions
Firefox 76
Replicable on Chrome 83.0.4103.61
</code></pre>

**Comment by WilcoFiers** [comment]
@harryhmc if you have a look at the WCAG techniques for bypass blocks, you can see there are other ways to satisfy this success criterion for WCAG, including using landmarks and headings:
https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=241#bypass-blocks

This is the accepted interpretation of WCAG. I can't speak for why WAVE is stricter than that, but I would argue that is a false positive.

Axe-core is highly configurable. You can do a number of things make this work:

1. Don't disable best practices in Cypress. Those are enabled by default. My recommendation would be to always use them.
2. You can change the tags of the rule in axe.configure()
3. You could explicitly turn this rule on while running axe-core, even if other best practices are disabled

**Comment by harryhmc** [comment]
I'll feed this back to my team - thanks for your quick response. :+1:
